microsoftgraph/microsoft-graph-explorer-v2
# Adds terms of use information bar

## Overview

Adds the 'Terms of use' information bar to the UI between the request and response windows. This has a 'close' button that dismisses this message.

### Demo
The information bar looks like below:

![image](https://user-images.githubusercontent.com/40403681/66301929-bbb53e80-e900-11e9-93ea-653df34160e5.png)


### Notes
This PR closes #168 

## Testing Instructions

* Log on to the tst environment.
* Notice the above terms of use message (from the screenshot above) placed in between the response and request windows.
* Close the message by clicking on the 'close' button at the top right corner.
